src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1597,7 +1597,7 @@ primitive_inst::primitive_inst(network & network, program_node const& node, bool
     , _inputs_memory_count(node.get_inputs_count())
     , _outputs_memory_count(node.get_outputs_count())
     , _fused_mem_count(node.get_fused_inputs_count())
-    , _fused_mem_offset((_fused_mem_count > 0 && node.has_fused_dep()) ? node.get_first_fused_dep_idx() : 0)
+    , _fused_mem_offset((_fused_mem_count > 0 && node.has_fused_dep() && node.get_first_fused_dep_idx() > 0) ? node.get_first_fused_dep_idx() : 0)
```

Comment:
`node.has_fused_dep()` check can be removed since it is already contained in `get_first_fused_dep_idx()`